service/src/test/java/bio/terra/tanagra/app/controller/QueryCountRegressionTest.java

```diff
@@ -1,11 +1,10 @@
-package bio.terra.tanagra.regression;
+package bio.terra.tanagra.app.controller;
```

Comment:
This is the regression test runner, not a test of any of the controller classes, so I don't think it should be the same package.

Reply:
reverted